docs/providers/aws/guide/functions.md

```diff
@@ -253,6 +253,29 @@ functions:
       TABLE_NAME: tableName2
 ```
 
+## KMS Key
+
+AWS Lambda uses [AWS Key Management Service (KMS)](https://aws.amazon.com/kms/) to encrypt your environment variables at rest. You can override the default KMS key used by the Lambda function at the provider or function level:
+
+```yml
+# serverless.yml
+service: service-name
+provider:
+  name: aws
+  kmsKeyArn: [your KMS key ARN]
```

Comment:
Would suggest to add an `environment` variable here because the KMS key only works together with an environment variable